R/tm_g_events_term_id.R

```diff
@@ -287,34 +280,39 @@ srv_g_events_term_id <- function(id,
           selected = choices[trt_index],
           choices = choices
         )
-      }
+      },
+      ignoreNULL = TRUE
     )
 
     output_q <- reactive({
       ANL <- data[[dataname]]() # nolint
 
-      validate(need(iv$is_valid(), "Misspecification error: please observe red flags in the encodings."))
-
-      validate(need(
-        is.factor(ANL[[input$arm_var]]),
-        "Selected arm variable needs to be a factor. Contact an app developer."
+      iv <- shinyvalidate::InputValidator$new()
+      iv$add_rule("term", shinyvalidate::sv_required(
```

Comment:
Should we add:

```
iv$add_rule("axis", shinyvalidate::sv_required("Please select an axis side")
```

Reply:
also for g_ae_overview ?

Reply:
Should we? If you select nothing, the labels still appear on the left. This might be a bug in the module, I don't know.